demos/linear-wave-equation/linear_wave_equation.py.rst

```diff
@@ -107,7 +107,7 @@ options at this point, we may either `lump` the mass, which reduces
 the inversion to a pointwise division::
 
       if lump_mass:
-          p += assemble(dt * inner(nabla_grad(v), nabla_grad(phi))*dx) / assemble(v*dx)
+          p += interpolate(assemble(dt * inner(nabla_grad(v), nabla_grad(phi))*dx) / assemble(v*dx), V)
```

Comment:
@connorjward can you update me on what assign is doing now. Do we still need this fix?

Reply:
I haven't made any recent changes to `assign`. I got quite confused when the adjoint got involved so stopped working on it.

Reply:
In which case can we accept this as a temporary fix? If so, I'll create an issue documenting the issue and tidy up this PR